### PR TITLE
Remove old code + fix indentation

### PR DESCRIPTION
**Pull request by DanielRakh**
"But why does have perfection in style important but not in the code base?" - 🍆

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -1,7 +1,72 @@
...
...k";font-size:6px;line-height:20px;text-align:center;color:#000;opacity:.25;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}.slick-dots li.slick-active button:before{color:#000;opacity:.75}.margin-auto,.line-break,.menu-line-break,.menu-container,.square-logo,.payment-instructions,.square-image,.item,.our-formula-line-break{margin:0 auto}@font-face{font-family:"Proxima-Light-Italic";src:url("../fonts/proxima-nova-light-italic.otf")}@font-face{font-family:"Proxima-Light";src:url("../fonts/proxima-nova-light.ttf") format("truetype")}@font-face{font-family:"Proxima-Regular";src:url("../fonts/ProximaNova-Regular.otf")}@font-face{font-family:"Proxima-Semi-Bold";src:url("../fonts/ProximaNova-Semibold.otf")}@font-face{font-family:"Proxima-Bold";src:url("../fonts/proxima-nova-bold.otf")}body{font:100% "Proxima-Light"}.temp-display{width:100%;text-align:center;background:#ff704d;color:#FFFFFF;box-sizing:border-box;padding:16px 0}.temp-display img{height:32px}.temp-display p{font-size:16px;letter-spacing:0.5px;margin-bottom:0;margin-top:10px}.temp-display-linebreak{border-bottom:1px solid white;width:35%;max-width:200px;margin:16px auto 8px auto}body{background:#FCFCFC}a{
```
Comment (DeProdigy) [inline css/main.css]: The `SASS` compilation resulted in an error and hence there is no `CSS` with this PR. Please recompile.

Comment (DeProdigy) [inline css/main.css]: I don't see that diff anymore. Weird.

**Comment by DeProdigy** [comment]
Same thing with two commits as before.

**Comment by DanielRakh** [comment]
Dafuq is going on?

**Comment by DeProdigy** [comment]
Cute PR name

**Comment by DanielRakh** [comment]
I think I know whats up. Hold on.

**Comment by DanielRakh** [comment]
I think we're good to go now...

**Comment by DeProdigy** [comment]
Can you squash these two into one?
Having an entire commit that removes two spaces seems excessive.

**Pull request merged**